### PR TITLE
Feature/add efile links

### PR DESCRIPTION
**Pull request by LindsayYoung**
adding the things for the front end

### CODE REVIEW

File: webservices/schemas.py [hunk 1]
```diff
@@ -394,7 +394,9 @@ class CandidateHistoryTotalSchema(schemas['CandidateHistorySchema'], schemas['Ca
         'ending_image_number': ma.fields.Str(),
         'pdf_url': ma.fields.Str(),
         'is_amended': ma.fields.Boolean(),
+        'document_description': ma.fields.Str(),
     },
```
Comment (jontours) [inline webservices/schemas.py]: To follow up with our conversation, just put this line after 397:

`'form_type': ma.fields.Str(),`.

There is some things that automagically happen (or don't) with the schemas.

Comment (LindsayYoung) [inline webservices/schemas.py]: `form_type` is defined in the model. It is already showing up for me locally.

**Comment by LindsayYoung** [comment]
closes https://github.com/18F/openFEC/issues/1887

File: webservices/schemas.py (hunk 1 above)
Comment (jontours) [inline webservices/schemas.py]: Yeah sorry, I think I confused myself when looking at this.  What was the field that you wished to expose but wasn't showing up?

**Comment by LindsayYoung** [comment]
@jontours do you mind reviewing this one when you have a chance?

File: webservices/schemas.py (hunk 1 above)
Comment (LindsayYoung) [inline webservices/schemas.py]: No worries, I was trying to make `document_discription`, and I needed to make `report_type_full` for that

File: webservices/common/models/filings.py [hunk 2]
```diff
@@ -74,18 +78,34 @@ class EFilings(db.Model):
...
     @property
     def is_amended(self):
         if self.superceded is not None:
             return True
         return False
 
+    @property
+    def pdf_url(self):
+        image_number = str(self.beginning_image_number)
+        return 'http://docquery.fec.gov/pdf/{0}/{1}/{2}.pdf'.format(image_number[-3:], image_number, image_number)
```
Comment (jontours) [inline webservices/common/models/filings.py]: These URLs are interesting.  But one thing I'd suggest is just use image number once, i.e. 
`return 'http://docquery.fec.gov/pdf/{0}/{1}/{1}.pdf'.format(image_number[-3:], image_number)`

But that's nitpicking really.

**Comment by jontours** [comment]
Besides that one comment looks good to go, tested locally as well.

File: webservices/common/models/filings.py (hunk 2 above)
Comment (LindsayYoung) [inline webservices/common/models/filings.py]: Thanks for catching that!

**Comment by jontours** [comment]
We'll probably want to merge in to dev if we wish to test these with the actual data.  I can do that tomorrow morning.

**Pull request merged**